lessons/specifics/poolboy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Poolboy
+category: specifics
+order: 1
```

Comment:
I agree on these having their own section and even probably putting a warning in it. When picking libraries there is going to be an obvious bias and though many things are 'standard' now the language is quite young. I imagine in the coming year we'll see some things along the lines of Hpricot --> Nokogiri becoming the HTML etc parser of choice like back in the early Ruby days. Its just natural.

That said, if we give a warning I think its reasonable and wont mislead anyone. And lessons like these provide real value.